login-workflow/example/src/components/CustomDetailsScreen.tsx

```diff
@@ -43,6 +43,7 @@ export const CustomDetailsScreen: React.FC<AccountDetailsFormProps> = (props) =>
                 onKeyPress={(e): void => {
                     if (e.key === 'Enter' && currencyRef.current) currencyRef.current.focus();
                 }}
+                style={{ marginTop: '32px' }}
```

Comment:
See my suggestion above that will eliminate the need for users to have to manually add these margins